src/components/EditingOverview/index.tsx

```diff
@@ -103,7 +113,14 @@ const EditingOverview = observer(() => {
                                     const user = userStore.find(userId);
                                     return (
                                         <div key={userId} className={clsx(styles.usersTasks)}>
-                                            <span className={styles.user}>{user!.nameShort}</span>
+                                            <div>
+                                                <LiveStatusIndicator
+                                                    size={0.3}
+                                                    userId={user!.id}
+                                                    className={clsx(styles.liveStatusIndicator)}
+                                                />
+                                                <span className={styles.user}>{user!.nameShort}</span>
```

Comment:
<img width="299" height="200" alt="Image" src="https://github.com/user-attachments/assets/2df2da86-902c-4dd6-affd-8f5089f625a3" />

should we add
```css
white-space: nowrap;
```

for `styles.user` here?